conformance_tests/sysman/test_sysman_pci/src/test_sysman_pci.cpp

```diff
@@ -171,9 +171,21 @@ TEST_F(PCI_TEST, GivenSysmanHandleWhenRetrievingPCIStatsThenStatsAreReturned) {
     lzt::destroy_command_queue(cq);
     lzt::destroy_command_list(command_list);
 
-    // delay to prevent identical counters in rare cases
-    std::this_thread::sleep_for(std::chrono::seconds(1));
-    auto pci_stats_later = lzt::get_pci_stats(device);
+    zes_pci_stats_t pci_stats_later{};
+    uint32_t wait = 0;
+    do {
+      pci_stats_later = lzt::get_pci_stats(device);
+      wait += IDLE_WAIT_TIMESTEP_MSEC;
+      // sleep for sometime before next check
+      std::this_thread::sleep_for(
+          std::chrono::milliseconds(IDLE_WAIT_TIMESTEP_MSEC));
+    } while (pci_stats_later.timestamp == pci_stats_initial.timestamp &&
+             wait <= IDLE_WAIT_TIMEOUT_MSEC);
+
+    if (wait > IDLE_WAIT_TIMEOUT_MSEC) {
```

Comment:
This doesn't seem correct. If TIMEOUT = n and TIMESTEP = 1, and PCI stats are available in the last iteration, the wait time will be n+1, which should not be considered as a failure."

Reply:
I wanted to consider the n+1 th iteration as failure. Expecting stats to be updated within n iterations (Timeout).
if in n+1 th iteration, eventhough the stats gets updated it has to be failure

Reply:
In the nth iteration itself, wait time will become n + 1, if valid stats are found in nth iteration, then also test will fail.